packages/manager/modules/vps/src/dashboard/migrate/migrate.html

```diff
@@ -0,0 +1,113 @@
+<oui-tile-definition
+    data-term="{{:: 'vps_dashboard_migrate_title'| translate}}"
+>
+    <oui-tile-description>
+        <oui-select-picker
```

Comment:
why use a select picker when user has nothing to select? this will have impact on accessibility. Consider creating a box that looks like select picker using select picker styles.